traffic_ops/traffic_ops_golang/ping/riak.go

```diff
@@ -21,24 +21,34 @@ package ping
 
 import (
 	"errors"
+	"fmt"
 	"net/http"
 
+	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/riaksvc"
 )
 
 func Riak(w http.ResponseWriter, r *http.Request) {
+	alerts := tc.CreateAlerts(tc.WarnLevel, fmt.Sprintf("This endpoint is deprecated, please use GET /vault/ping instead"))
```

Comment:
This should also specify the API version, since it has to change in order for the alternative to be used.

Reply:
done